submissions/tests/test_team_api.py

```diff
@@ -237,6 +237,19 @@ def test_get_team_submission(self):
             TeamSubmissionSerializer(team_submission_model).data
         )
 
+    def test_get_team_submission_from_individual_submission(self):
```

Comment:
Is there a case where we would expect the exception case to happen? If so, it might also use a test case.

Reply:
No specific case. This is for defensive purposes.

Reply:
i'd probably agree with nathan, probably best for completeness' sake to put a teensy test in that hits that use case